### PR TITLE
Finished Login design

### PR DESCRIPTION
**Pull request by TinelValentin**

### CODE REVIEW

**Review by BrassoiCatalin — APPROVED** [review]

File: SevenSlots/SevenSlots/SevenSlots/View/SignInView.xaml [hunk 1]
```diff
@@ -4,30 +4,97 @@
              xmlns:sevenslots1="clr-namespace:SevenSlots.View" 
              x:DataType="sevenslots1:SignInView"
              x:Class="SevenSlots.View.SignInView"
-             BackgroundColor="#523943">
+             xmlns:controls="clr-namespace:SevenSlots.Controls;assembly=SevenSlots"
+             xmlns:xct="http://xamarin.com/schemas/2020/toolkit"
+             >
+    <ContentPage.Background>
+        <LinearGradientBrush StartPoint="0, 0" EndPoint="1, 1">
+            <GradientStop Color="#802525"  Offset="0.0"/>
```
Comment (Sorin37) [inline SevenSlots/SevenSlots/SevenSlots/View/SignInView.xaml:12]: Two spaces between Color and Offset

File: SevenSlots/SevenSlots/SevenSlots/View/SignInView.xaml [hunk 2]
```diff
@@ -4,30 +4,97 @@
...
-                     VerticalOptions="Center"
-                     HorizontalOptions="Center">
-            <Button Text="LogIn" Style="{StaticResource ButtonSmootherStyle}"/>
-            <Button Text="Sign In" Style="{StaticResource ButtonSmootherStyle}"/>
-            <Button Text="Forgot Password" Style="{StaticResource ButtonSmootherStyle}"/>
-        </StackLayout>
+        <Image Source="casinoBackground4.jpg" WidthRequest="400" Margin="-10,0,0,0" Opacity="0.6">
+            <Image.Clip>
+                <EllipseGeometry                                
+                    Center="205,0"
```
Comment (Sorin37) [inline SevenSlots/SevenSlots/SevenSlots/View/SignInView.xaml:35]: no space after comma

File: SevenSlots/SevenSlots/SevenSlots/View/SignInView.xaml [hunk 3]
```diff
@@ -4,30 +4,97 @@
...
+        <StackLayout HorizontalOptions="Center" VerticalOptions="Center">
+            <Frame HeightRequest="380" WidthRequest="300" CornerRadius="20" BorderColor="Transparent">
+            <Frame.Resources>
+                <Style TargetType="{x:Type Frame}">
+                    <Setter Property="Background">
+                        <Setter.Value>
+                            <LinearGradientBrush StartPoint="1, 0"
+                                                        EndPoint="1, 1">
+                                <GradientStop Color="#4f0a0a" 
+                                                            Offset="0.0"/>
```
Comment (Sorin37) [inline SevenSlots/SevenSlots/SevenSlots/View/SignInView.xaml:49]: ugly indentation

File: SevenSlots/SevenSlots/SevenSlots/View/SignInView.xaml [hunk 4]
```diff
@@ -4,30 +4,97 @@
...
+                                <GradientStop Color="#b04343"
+                                                            Offset="0.7"/>
+                                <GradientStop Color="#e36f6f"
+                                                            Offset="1.0"/>
+                            </LinearGradientBrush>
+                        </Setter.Value>
+                    </Setter>
+                </Style>
+            </Frame.Resources>
+            <Grid HeightRequest="380" WidthRequest="340" >
```
Comment (Sorin37) [inline SevenSlots/SevenSlots/SevenSlots/View/SignInView.xaml:61]: extra space before tag end

File: SevenSlots/SevenSlots/SevenSlots/View/SignInView.xaml [hunk 5]
```diff
@@ -4,30 +4,97 @@
...
+                <Grid.RowDefinitions>
+                    <RowDefinition Height="150"/>
+                    <RowDefinition Height="Auto"/>
+                    <RowDefinition Height="Auto"/>
+                </Grid.RowDefinitions>
+                <Image Source ="user.png" WidthRequest="100" HorizontalOptions="Center" Grid.Row="0" Margin="0,-50,0,0"/>
```
Comment (Sorin37) [inline SevenSlots/SevenSlots/SevenSlots/View/SignInView.xaml:67]: no space after comma

File: SevenSlots/SevenSlots/SevenSlots/View/SignInView.xaml [hunk 6]
```diff
@@ -4,30 +4,97 @@
...
+                <Frame Grid.Row="1" Style="{StaticResource InputFrame}"
+                       xct:ShadowEffect.Color="White"
+                       xct:ShadowEffect.Radius="10"
+                       x:Name="SSNFrame">
+                    <controls:CustomInputBox
+                        HeightRequest="55"
+                        WidthRequest="300"
+                        x:Name="SSNInput" VerticalOptions="FillAndExpand"
+                        Placeholder="Username"
+                        PlaceholderColor="DarkGray" >
```
Comment (Sorin37) [inline SevenSlots/SevenSlots/SevenSlots/View/SignInView.xaml:77]: extra space before end of tag

File: SevenSlots/SevenSlots/SevenSlots/View/SignInView.xaml [hunk 7]
```diff
@@ -4,30 +4,97 @@
...
+                        HeightRequest="55"
+                        WidthRequest="300"
+                        x:Name="PasswordInput" VerticalOptions="FillAndExpand"
+                        Placeholder="Password"
+                        PlaceholderColor="DarkGray" >
+                    </controls:CustomInputBox>
+                </Frame>
+            </Grid>
+        </Frame>
+        <Button  WidthRequest="180" HeightRequest="50" Margin="0,-40,0,0" HorizontalOptions="Center"
```
Comment (Sorin37) [inline SevenSlots/SevenSlots/SevenSlots/View/SignInView.xaml:93]: no space after comma

File: SevenSlots/SevenSlots/SevenSlots/View/SignInView.xaml [hunk 8]
```diff
@@ -4,30 +4,97 @@
...
+                CornerRadius="30" Text="Login" FontSize="20" FontAttributes="Bold" xct:ShadowEffect.Color="Black"
+                xct:ShadowEffect.Radius="10" Background="#3d1313" TextColor="White"/>
+        <Button WidthRequest="200" HeightRequest="50" Margin="0,0,0,0" HorizontalOptions="Center"
```
Comment (Sorin37) [inline SevenSlots/SevenSlots/SevenSlots/View/SignInView.xaml:96]: no space after comma

File: SevenSlots/SevenSlots/SevenSlots/View/SignInView.xaml [hunk 9]
```diff
@@ -4,30 +4,97 @@
...
+                CornerRadius="30" Text="Forgot Password" FontSize="15" FontAttributes="Bold" Background="Transparent" TextColor="White"/>
+    </StackLayout>
```
Comment (Sorin37) [inline SevenSlots/SevenSlots/SevenSlots/View/SignInView.xaml:98]: both stack layouts are on the same column

**Review by Sorin37 — CHANGES_REQUESTED** [review]

**Pull request merged**